subsys/bluetooth/host/gatt.c

```diff
@@ -2130,23 +2138,37 @@ static int gatt_notify_mult(struct bt_conn *conn, uint16_t handle,
 	}
 
 	if (!*buf) {
-		*buf = bt_att_create_pdu(conn, BT_ATT_OP_NOTIFY_MULT,
-					 sizeof(*nfy) + params->len);
+		*buf = bt_att_create_pdu(conn, BT_ATT_OP_NOTIFY_MULT, pdu_len);
 		if (!*buf) {
 			return -ENOMEM;
 		}
 
-		bt_att_set_tx_meta_data(*buf, params->func, params->user_data);
+		bt_att_set_tx_meta_data(*buf, tuple[0].func, tuple[0].user_data);
```

Comment:
What if the `func` of each `bt_gatt_notify_params` is different?

Reply:
Here should add one assert, it must be the same, w.r.t this us a tuple of data